test/e2e/actions/deploy/deploy_operator.go

```diff
@@ -106,9 +128,109 @@ func MultiNamespaceOperator(data *model.TestDataProvider, watchNamespace []strin
 		kustomOperatorPath := data.Resources.GetOperatorFolder() + "/final.yaml"
 		utils.SaveToFile(kustomOperatorPath, kustomize.Build(data.Resources.GetOperatorFolder()))
 		kubecli.Apply(kustomOperatorPath)
-		Eventually(
-			kubecli.GetPodStatus(config.DefaultOperatorNS),
-			"5m", "3s",
-		).Should(Equal("Running"), "The operator should successfully run")
+		CheckOperatorRunning(data, config.DefaultOperatorNS)
+	})
+}
+
+func DeleteProject(testData *model.TestDataProvider) {
+	By("Delete Project", func() {
+		projectId := testData.Project.Status.ID
+		Expect(testData.K8SClient.Get(testData.Context, types.NamespacedName{Name: testData.Project.Name, Namespace: testData.Project.Namespace}, testData.Project)).Should(Succeed(), "Get project failed")
+		Expect(testData.K8SClient.Delete(testData.Context, testData.Project)).Should(Succeed(), "Delete project failed")
+		aClient := atlas.GetClientOrFail()
+		Eventually(func() bool {
```

Comment:
Potentially missing `gomega` context. The `aClient.IsProjectExists` is using `Fail` handler

Reply:
Done